Definitions.md

```diff
@@ -0,0 +1,16 @@
+#Definition Of Terms
+
+##Job
```

Comment:
@ntijanic Good point. We could just amend the `Workflow` text to say it can contain workflows or tasks. I don't think we need to worry about concepts like supertypes in a thing like this. In terms of implementation I agree with you though - if the text were changed as I just described and I were going to implement it that's how I'd do it.